packages/block-library/src/columns/edit.js

```diff
@@ -78,11 +79,15 @@ function ColumnsEditContainer( {
 	const blockProps = useBlockProps( {
 		className: classes,
 	} );
-	const innerBlocksProps = useInnerBlocksProps( blockProps, {
-		allowedBlocks: ALLOWED_BLOCKS,
-		orientation: 'horizontal',
-		renderAppender: false,
-	} );
+	const ariaDescribedById = sprintf( '%s-description', clientId );
```

Comment:
Do we really need `sprintf` here? 🤔 I believe we can accomplish the same result concatenating with backticks.
```suggestion
	const ariaDescribedById = `${ clientId }-description`;
```